### DISCUSSION TITLE
Network Indirect Message Timeout and Packet Timing

### DISCUSSION THREAD

**Discussion by Aimen-Hammou** [Koenkk/zigbee2mqtt#13161] [General]
Hi,

I would like to know how much time is set the dongle to hold the packet before discard it with a MAC Transaction Expired.

I remember an old firmware with like 5 minute timer.

Looking at the ZNP.syscfg the maximum value of "Network Indirect Message Timeout" is 127 seconds.

**Comment by Aimen-Hammou** [comment]
@Koenkk I hate to disturb you, but how did you accomplish that? How much is the current timer value?

**Reply by Koenkk** [reply]
It is set to 7 seconds:

<img width="697" alt="Screenshot 2022-07-15 at 17 26 54" src="https://user-images.githubusercontent.com/2892853/179255461-7d26e3f9-ede4-4d47-832a-adb59e8c6c4a.png">

**Reply by Aimen-Hammou** [reply]
@Koenkk I remember old firmware had like 5 minutes before receiving MAC Transaction Expired. The Current Value is 7? Isn't it source for many problems with such low value?

**Reply by Koenkk** [reply]
It was never 5 minutes, maybe 5 seconds.
